include/vapor/GeoImageTMS.h

```diff
@@ -27,8 +30,14 @@ class RENDER_API GeoImageTMS : public GeoImage {
  GeoImageTMS();
  virtual ~GeoImageTMS();
 
+ static bool IsTMSFile( std::string path );
+ static std::string TilePath( std::string dir, size_t tileX, size_t tileY, int lod );
```

Comment:
dir -> 'file' or 'path'. 'dir' would imply directory,  wouldn't it, which the argument is not

Reply:
The 'dir' argument in TilePath can either be a file or a directory.  'file' describes both, so I'll change it to that.